Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -49,21 +56,19 @@ class FSquare
 			std::cout << "Invalid input, please enter The side of Your square again" << std::endl;
 			std::cin.clear();
 			std::cin.ignore(1000, '\n');
-			std::cin >>EntrySide;
+			std::cin >> EntrySide;
 		}
 		SideLenght = EntrySide;
 	}
-	float GetAreaSquare()
+	const float GetAreaSquare()
```

Comment:
This syntax says that the GetAreaSquare function is returning a `const float` which is not desirable, you shouldn't return const values from functions. Instead, what you want to do is make the method `const` and for that, you want to type const at the _end_ of the function declaration:
```
float GetAreaSquare() const
{

}
```

Reply:
changed and understanded